libs/template/unpatch/src/lib/unpatch.directive.ts

```diff
@@ -116,12 +116,7 @@ export function unpatchEventListener(
  *
  * @publicApi
  */
-// eslint-disable-next-line @angular-eslint/directive-selector
 @Directive({ selector: '[unpatch]', standalone: true })
-/**
- * @todo: add prefix [rxUnpatch]
- */
```

Comment:
we aren't going to be making changes in API, so I thought we should remove the comment for todo.